custom_components/unifiprotect/config_flow.py

```diff
@@ -96,6 +139,19 @@ async def async_step_user(self, user_input=None):
             },
         )
 
+    async def _show_reauth_form(self, errors=None):
```

Comment:
```suggestion
    async def _async_show_reauth_form(self, errors=None):
```

Anything that should run in async context should be prefixed with `async`.  Its a normal function decorate with `@callback` from `homeassistant.core` so Home Assistant internals know to run it in the event loop and not the executor